src/entities/update-entities.js

```diff
@@ -16,17 +16,18 @@ const shortNames = require('./shortNames.js')
 const whoisRegexArray = whoisIgnoreList.map(regex => new RegExp(regex, 'gi'))
 const SSLRegexArray = SSLIgnoreList.map(regex => new RegExp(regex, 'gi'))
 const currentDate = new Date(Date.now()).toISOString()
+const WHOIS_XML_TOKEN = ''// credentials
```

Comment:
This wasn't defined - I'm guessing you are adding this line manually @dharb? Is this OK if I define it here like this?

Reply:
Yeah we use our own access token for the api. Works for me 👍